cdhweb/resources/views.py

```diff
@@ -44,7 +46,12 @@ class Homepage(TemplateView):
     template_name = 'site_index.html'
 
     def get_context_data(self, *args, **kwargs):
-        # TODO: highlighted/featured blog post
+        # get featured posts if there are any
+        if (BlogPost.objects.featured().exists()):
+            updates = BlogPost.objects.featured()
```

Comment:
This _looks_ inefficient to me, although I think it probably actually isn't (because Django is clever).  However, I think it would be cleaner and more readable to do something like:
```
updates = BlogPost.objects.featured()
if not updates.exists():
   updates = BlogPost.objects.recent()[:3]
```
Even if there are featured items, don't we still need `recent()` and `[:3]` ?

In that case, you might do something like this:
```
updates = BlogPost.objects.featured()
if not updates.exists():
   updates = BlogPost.objects.all()
updates = updates.recent()[:3]
```